package.json

```diff
@@ -788,7 +788,7 @@
     "deepmerge": "^4.2.2",
     "del": "^6.1.0",
     "elastic-apm-http-client": "^11.0.1",
-    "elastic-apm-node": "^3.45.0",
+    "elastic-apm-node": "git+https://github.com/afharo/apm-agent-nodejs.git#add-agent.setGlobalLabel",
```

Comment:
This can't be merged until https://github.com/elastic/apm-agent-nodejs/pull/3337 is merged and published

Reply:
@david-luna can you help us with this?